R/data_color.R

```diff
@@ -471,13 +470,15 @@ rgba_to_hex <- function(colors) {
       byrow = TRUE
     )
 
-  alpha_vals <- color_matrix[, "alpha"] %>% unname()
+  alpha <- color_matrix[, "alpha"] %>% unname()
 
+  # Convert color matrix to hexadecimal colors in the #RRGGBBAA format
   colors_to_hex <-
-    farver::encode_colour(
-      colour = color_matrix,
-      from = "rgb",
-      alpha = alpha_vals
+    grDevices::rgb(
+      red = color_matrix[, "r"] / 255,
+      green = color_matrix[, "g"] / 255,
+      blue = color_matrix[, "b"] / 255,
+      alpha = alpha
```

Comment:
maxColorValue = 255, and stop dividing red/green/blue by 255, and multiply alpha by 255